package.json

```diff
@@ -8,13 +8,15 @@
     "@octokit/webhooks": "^6.1.0",
     "bcryptjs": "^2.4.3",
     "concurrently": "^4.1.0",
+    "cookie-session": "^1.3.3",
```

Comment:
Previous iterations had missing packages, resulting in them not working correctly